src/NServiceBus.RabbitMQ/Configuration/RabbitMQTransportSettingsExtensions.cs

```diff
@@ -64,5 +64,17 @@ public static TransportExtensions<RabbitMQTransport> TimeToWaitBeforeTriggeringC
             transportExtensions.GetSettings().Set(SettingsKeys.TimeToWaitBeforeTriggeringCircuitBreaker, waitTime);
             return transportExtensions;
         }
+
+        /// <summary>
+        /// Overrides the default usePublisherConfirms or value set in connection string.
+        /// </summary>
+        /// <param name="transportExtensions"></param>
+        /// <param name="publisherConfirms">Controls if publisher confirms should be used.</param>
+        /// <returns></returns>
+        public static TransportExtensions<RabbitMQTransport> UsePublisherConfirms(this TransportExtensions<RabbitMQTransport> transportExtensions, bool publisherConfirms)
```

Comment:
Do we have a precedent for how Boolean transport extension settings are handled?

Specifically, I'm wondering if `UsePublisherConfirms()` (parameterless) would be preferable to `UsePublisherConfirms(true)`. Would there ever be a reason to call `UsePublisherConfirms(false)`?

Come to think of it, what is the default when the setting is not specified? True or false?